packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -286,6 +286,16 @@ export default class VTable {
                     });
                 }
                 break;
+
+            case TableOperation.AlignCenter:

```

Comment:
Please test the feature in OWA:
1. New compose
2. Insert a table, center the table
3. Send it out
4. When receive, see if the table is rendered correctly, read in both OWA and Outlook Desktop
If everything works well, we can go with this.

Reply:
![Table Aligment](https://user-images.githubusercontent.com/87443959/129799491-b4e55027-6710-4e82-8e60-d48033d1ffdc.gif)

Reply:
This looks nice. Please also try send the email out after align table to right, and check the receiving result, see if it still looks correctly.

Reply:
I tested in OWA and Outlook Desktop. I didn't find any bug in OWA, but for Outlook desktop, if I send a email with more than one table the second table loses the formatting 

![image](https://user-images.githubusercontent.com/87443959/129802311-04db2d2c-d225-4f4b-8d83-4162bafb9705.png)

Reply:
Even if it is not aligned 
![image](https://user-images.githubusercontent.com/87443959/129802399-ca68ad83-c3b9-4a33-a244-fe72dddeface.png)

Reply:
It works fine, if the cells are filled.
![image](https://user-images.githubusercontent.com/87443959/129802493-9c1efd27-531e-4f81-9bf9-d63600185ff0.png)